From 0c00db07192408291b71ab9991b91da3e96b1492 Mon Sep 17 00:00:00 2001
From: Michael Trensch <mtrensch@hilscher.com>
Date: Wed, 25 May 2022 10:13:15 +0200
Subject: [PATCH 2/2] Allow docker-readonly group to access docker socket.

Permissions will be determined / granted by opa-docker-authz plugin
---
 contrib/init/systemd/docker.socket | 1 +
 1 file changed, 1 insertion(+)

diff --git a/contrib/init/systemd/docker.socket b/contrib/init/systemd/docker.socket
index c76a23296e..5bf8c2af4f 100644
--- a/contrib/init/systemd/docker.socket
+++ b/contrib/init/systemd/docker.socket
@@ -5,6 +5,7 @@ Description=Docker Socket for the API
 # If /var/run is not implemented as a symlink to /run, you may need to
 # specify ListenStream=/var/run/docker.sock instead.
 ListenStream=/run/docker.sock
+ExecStartPost=/usr/bin/setfacl -m g:docker-readonly:rw- /run/docker.sock
 SocketMode=0660
 SocketUser=root
 SocketGroup=docker
-- 
2.25.1

